### PR TITLE
Disables Kilo due to frequent crashes

### PR DESCRIPTION
**Pull request by Cenrus**
/tg/ has reported the same problem and disabled the map accordingly. Kilo will return to active map rotation once they figure out how to fix the crashing.

:cl:
config: Temporarily disables KiloStation due to frequent crashing.
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Remove the votable tag instead

**Comment by qwertyquerty** [comment]
@Crossedfall configs

**Comment by Crossedfall** [comment]
Sad.

File: config/Sage/maps.txt [hunk 1]
```diff
@@ -50,5 +50,4 @@ endmap
 
 map kilostation
 	maxplayers 60
```
Comment (Crossedfall) [inline config/Sage/maps.txt:52]:
```suggestion
	maxplayers 60
	disabled
```

`votable` is technically fine to remain but removing it does no harm. That being said, you need to define the map as disabled for it to be removed from rotation.

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Once the disabled option is added this can be merged. I've already applied the config changes to Sage.

**Comment by qwertyquerty** [comment]
@Crossedfall

**Pull request merged**